src/styles/main.scss

```diff
@@ -1,3 +1,57 @@
+@import "./utils/variables";
+@import "./utils/functions";
+
 body {
+  line-height: 100px;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
   margin: 0;
 }
+
+.calendar {
+  margin: 50px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  width: width-calculate($day-size, $gap, $border-size);
+
+  &__day {
+    height: $day-size;
+    width: $day-size;
+    box-sizing: border-box;
+    text-align: center;
+    background-color: #eee;
+    border: $border-size solid black;
+    transition: 0.3s;
+
+    &:hover {
+      cursor: pointer;
+      background-color: #ffc0cb;
+      transform: translateY(-20px);
+    }
+  }
+
+  &:first-child {
+    margin-left: 500px;
+  }
+
+  @for $i from 1 through $month-length {
+    &__day--#{$i}::before {
+      content: "#{$i}";
+    }
+  }
```

Comment:
[checklist](https://github.com/mate-academy/layout_calendar/blob/master/checklist.md) 4th item, it works for loops too